src/Core/Settings.cpp

```diff
@@ -4383,7 +4383,8 @@ Possible values:
 Allow sharing set objects build for IN subqueries between different tasks of the same mutation. This reduces memory usage and CPU consumption
 )", 0) \
     DECLARE(Bool, use_query_condition_cache, false, R"(
-Enable the query condition cache.
+Enable the query condition cache. The cache stores ranges of granules in data parts which satisfy the condition in `WHERE` clause,
```

Comment:
@Blargian Looking at https://clickhouse.com/docs/operations/query-condition-cache

```
[...]
The information is recorded as a single bit: a 0 bit represents that no row matches the filter whereas a 1 bit means that at least one matching row exists.
[...]
For example, the first execution of query [...] will store ranges of the table which do not satisfy the predicate.
[...]
```

... this should read "which do not satisfy the condition". I know, that's slightly unintuitive but it's the way the cache works.